packages/test-utils/package.json

```diff
@@ -37,6 +37,8 @@
     "vue-template-compiler": "^2.x"
   },
   "dependencies": {
-    "lodash": "^4.17.4"
+    "html-tags": "^2.0.0",
+    "lodash": "^4.17.4",
+    "svg-elements": "^1.0.2"
```

Comment:
both libs are just json lists. an alternative would be just copy paste the json lists into the project

Reply:
IMHO,
Since their size is small,
I think it is not necessary to use library.
`Vue.js` does not use the library as below.
https://github.com/vuejs/vue/blob/f76d16ed9507d4c2a90243ea3d77ccf00df29346/src/platforms/web/util/element.js#L11-L32

Reply:
yeah, I agree. Just going to grab the list from vuejs/vue repo.